dissect/target/filesystems/dir.py

```diff
@@ -54,7 +54,8 @@ def _resolve(self):
         return self
 
     def get(self, path):
-        return self.fs.get(fsutil.join(self.path, path))
+        path = fsutil.join(self.path, path, alt_separator=self.fs.alt_separator)
```

Comment:
Should the `DirLoader` not set the `alt_separator` of the `DirectoryFilesystem` to `\\` if it's Windows?

Reply:
It would make it less surprising for users that they couldn't use `\`es on a filesystem that looks Windows-y. Functionally it wouldn't matter as `DirFilesystem` is an abstraction on top of the actual filesystem.